ambari-server/src/main/resources/common-services/HBASE/0.96.0.2.0/metrics.json

```diff
@@ -242,7 +242,7 @@
               "temporal": true
```

Comment:
I think this file is not related. It's ancient `common-services/HBASE/0.96.0.2.0` that is shared between a few stacks. The actual metadata is inherited from `stacks/HDP/2.3/services/HBASE`